packages/fx-core/src/plugins/resource/bot/index.ts

```diff
@@ -185,6 +191,42 @@ export class TeamsBot implements Plugin {
     );
   }
 
+  public async getQuestions(
+    stage: Stage,
+    context: PluginContext
+  ): Promise<Result<QTreeNode | undefined, FxError>> {
+    Logger.setLogger(context.logProvider);
+
+    if (stage === Stage.create) {
+      const result = await TeamsBot.runWithExceptionCatching(
```

Comment:
just return it, the local variable is redundant

Reply:
fixed